packages/react-router/lib/components.tsx

```diff
@@ -130,23 +131,62 @@ export function mapRouteProperties(route: RouteObject) {
   return updates;
 }
 
+export interface MemoryRouterOpts {
```

Comment:
Extracted to an interface so it gets it's own docs page and to align with `DOMRouterOpts`